.spotbugs-check.xml

```diff
@@ -25,7 +25,6 @@
         <Confidence value="3" /> <!-- 3 is low priority, 2 is medium priority, 1 is high priority -->
         <Or>
            <Bug pattern="IS2_INCONSISTENT_SYNC" />
-           <Bug pattern="LI_LAZY_INIT_STATIC" />
```

Comment:
It looks like this is removing LI_LAZY_INIT_STATIC (in the spotbugs config), not SLF4J_FORMAT_SHOULD_BE_CONST (in the title)

Reply:
Wrong title, oops.